mmv1/products/compute/RegionResizeRequest.yaml

```
@@ -0,0 +1,413 @@
+  # Link to the REST API reference for the resource. For example,
+  # https://cloud.google.com/alloydb/docs/reference/rest/v1/projects.locations.backups
+  api: 'https://cloud.google.com/compute/docs/reference/rest/v1/instanceGroupManagerResizeRequests'
+docs:
+### List Method ###
+base_url: 'projects/{{project}}/regions/{{region}}/instanceGroupManagers/{{instance_group_manager}}/resizeRequests'
+### Get Method
+self_link: 'projects/{{project}}/regions/{{region}}/instanceGroupManagers/{{instance_group_manager}}/resizeRequests/{{name}}'
+immutable: true
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+# Sets parameters for handling operations returned by the API.
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  # Overrides which API calls return operations. Default: ['create',
+  # 'update', 'delete']
+  # actions: ['create', 'update', 'delete']
+  operation:
+    base_url: '{{op_id}}'
+    kind: 'compute#operation'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'targetLink'
+    resource_inside_response: false
+  error:
+    path: 'error/errors'
+    message: 'message'
+### Update method ###
+# Resize requests are currently not update-able
+
+### Delete Method ###
+# Custom delete method to handle resize request cancellations vs. deletions.
+# If a resize request is in the CREATING or ACCEPTED state, it must be canceled before it can be
+# deleted. If a resize request is NOT in any of the mentioned state, it can be directly deleted.
+custom_code:
+  custom_delete: 'templates/terraform/custom_delete/compute_rmig_resize_request_delete.go.tmpl'
+# Examples for testing
+examples:
+  - name: 'compute_rmig_resize_request'
+    min_version: beta
+    primary_resource_id: 'a3_resize_request'
+    vars:
+      resize_request_name: 'a3-dws'
+# Resize request parameters injected via URL
+parameters:
+  - name: 'region'
+    type: ResourceRef
+    description: |
+      Name of the compute region scoping this request. Name should conform to RFC1035.
+    url_param_only: true
+    required: true
+    resource: 'Region'
+    imports: 'name'
+  - name: 'instanceGroupManager'
+    type: ResourceRef
+    description: |
+      The name of the regional managed instance group. The name should conform to RFC1035 or be a resource ID.
+      Authorization requires the following IAM permission on the specified resource instanceGroupManager:
+      *compute.instanceGroupManagers.update
+    url_param_only: true
+    required: true
+    resource: 'InstanceGroupManager'
+    imports: 'name'
+# Non-URL parameters including input and output parameters
+properties:
+  - name: 'creationTimestamp'
+    type: Time
+    description: |
+      The creation timestamp for this resize request in RFC3339 text format.
+    output: true
+  - name: 'state'
+    type: Enum
+    description: |
+      [Output only] Current state of the request.
```

```suggestion
      Current state of the request.
```

This text will be auto-generated in the docs.

---

Removed.